### ISSUE TITLE
Steam Input doesn't work with some games in Big Picutre mode

### ISSUE THREAD

**Issue by cold-distance** [flathub/com.valvesoftware.Steam#1369]
Hi, I detected that Steam Input doesn't work with some games if Big Picture is enabled, but it works with the Steam client in desktop mode.

Some of the games affected are the native version for Linux of BallisticNG and Split/Second (this one is only for Windows). I use Proton 9.0-3 and the Steam version is 1731433018 obtained from the Flatpak launcher. My operating system is now Fedora Silverblue 41 with the Wayland session on GNOME 47.1.

I tried a DualShock 4 and an 8Bitdo Ultimate 2.4G Wireless and I have the same result. This is my first the day with the 8BitDo controller, but I'm very sure that I could use my DualShock 4 with Steam Input in Big Picture mode previously with BallisticNG and Split/Second. I prefer to use Big Picture because I usually play with a controller.

I see that the version of steam-devices provided by RPM Fusion is 1.0.0.82, while the Flatpak launcher is in the version 1.0.0.81.

![Captura desde 2024-11-29 00-04-37](https://github.com/user-attachments/assets/6d88a006-6982-42c4-b9fe-3db03840b2ad)

**Comment by Etaash-mathamsetty** [comment]
Heroic Games Launcher has the same issue, (Lutris maybe as well)

**Comment by cold-distance** [comment]
OK. I detected the problem with the native version for Linux of Huntdown. https://store.steampowered.com/app/598550/HUNTDOWN/

If downgrading to freedesktop 23.08 solves the problem, I hope to see this version of Steam following the same steps.

**Comment by Etaash-mathamsetty** [comment]
@bbhtt friendly ping because it's a rather serious issue imo

**Comment by bbhtt** [comment]
Downgrading is not a solution, we can't permanently keep it on a runtime that will be EOL in an year without any solution. 

This was reported to the runtime as well https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/issues/1818 but none of us has access to controllers or steam decks to debug the issue.

I'd defer to @smcv for insights on this, since they are involved with valve.

**Comment by bbhtt** [comment]
If it helps I can make master 23.08 for now and beta with 24.08 but switching master to 24.08 is an inevitability.

**Comment by bbhtt** [comment]
There should be some controller log files called `controller_ui.txt` or `controller.txt` somewhere under `~/.var/app/com.valvesoftware.Steam`. Can anyone provide those?

**Comment by Etaash-mathamsetty** [comment]
> > I tried a DualShock 4 and an 8Bitdo Ultimate 2.4G Wireless and I have the same result.
> 
> Is only wireless controllers affected?

It was an issue with every controller wired or wireless when I tested it with heroic

**Comment by Etaash-mathamsetty** [comment]
> There should be some controller log files called `controller_ui.txt` or `controller.txt` somewhere under `~/.var/app/com.valvesoftware.Steam`. Can anyone provide those?

I will install flatpak steam on my other machine soon to test

**Comment by bbhtt** [comment]
Please test the builds in https://github.com/flathub/com.valvesoftware.Steam/pull/1377, https://github.com/flathub/com.valvesoftware.Steam/pull/1376 and say which one is working. If the link is 404, you can comment `bot, build` to start a new one.

**Comment by guihkx** [comment]
I'm interested to test this, because my DualShock 4 controller still works just fine with the few Steam games I have installed. 

Unfortunately, I don't own any of the games that are broken, so is there a known free to play game (preferably lightweight too), that also has the issue for me to test it?

My DS4 (connected via USB) still works as expected with the following games (24.08 runtime + Big Picture):

- [Terraria](https://store.steampowered.com/app/105600/Terraria/) (native)
- [Counter-Strike 2](https://store.steampowered.com/app/730?snr=2_9_100000__apphubheader) (native)
- [Plants vs. Zombies GOTY Edition](https://store.steampowered.com/app/3590/Plants_vs_Zombies_GOTY_Edition/) (Proton 9; requires a controller preset from the community)

**Comment by Etaash-mathamsetty** [comment]
yeah it was working in my testing too, I will try heroic on 24.08 soon and see if it works there. I dunno what happened

**Comment by cold-distance** [comment]
1738026274 version of the Steam Client and the bug is still there with Fedora Silverblue 41 fully updated.

I recently bought a GuliKit KK3 Pro that is identified as a generic Xbox 360 controller with its official dongle, so now I can disable Steam Input to play all compatible games with a controller, but with Steam Input the problem even is getting worse because games like #DRIVE Rally (native Unity) also doesn't work.
